docs/operations/advanced/deployment.md

```diff
@@ -0,0 +1,128 @@
+# Alternative Deployment Patterns
+
+**Reference guide for alternative KMS deployment approaches.**
+
+> **Production Deployments**: Use [Production Deployment Guide](../production-deployment.md) with official [Terraform MPC modules](https://github.com/zama-ai/terraform-mpc-modules) and KMS Helm charts (`charts/kms-core/`).
+
+This guide provides **references and concepts** for alternative deployment scenarios.
+
+## Deployment Options
+
+### Production (Recommended)
+- **Guide**: [Production Deployment](../production-deployment.md)
+- **Infrastructure**: [Terraform MPC Modules](https://github.com/zama-ai/terraform-mpc-modules)
+- **Application**: KMS Helm Charts (`charts/kms-core/`)
+- **Context**: 13-party threshold network, single-party management
+
+### Development & Testing
+- **Local Kubernetes**: Use `charts/kms-core/values-example-local.yaml`
+- **Configuration**: See [Configuration Guide](../configuration.md) for TOML examples
+- **Docker Development**: See Docker setup examples below
+
+### Custom Infrastructure
+- **Manual Kubernetes**: Adapt `charts/kms-core/templates/` for custom deployments
+- **Cloud Providers**: Use Terraform modules as reference for other clouds
+- **Bare Metal**: Extract configuration patterns from Helm values
+
+## Critical Requirements
+
+> **Complete Requirements**: See [Production Deployment Guide](../production-deployment.md#critical-requirements) for detailed infrastructure, security, and network requirements.
+
+### Quick Reference
+- **AWS Instance**: `c7a.16xlarge` (Nitro Enclaves required)
+- **Memory**: 16Gi minimum (32Gi recommended) 
+- **Ports**: 50100 (gRPC), 50001 (P2P), 9646 (metrics)
```

Comment:
Should we change the port reference to mention the expected port depends on the id. Or is that actually only in the test deployment we do that?